src/script4.R

```diff
@@ -0,0 +1,114 @@
+# authors: Fatime Selimi, Nicholas Wu, Tanmay Sharma, Neel Phaterpekar
+# date: 2019-12-28
+
+"Performs statistical analysis on preprocessed data and save figures and tables 
+into new file.
+
+Usage: script4.R --data=<data> --sum_data=<sum_data> --out_dir=<out_dir>
+ 
+Options:
+--data=<data>          Path to the file containing the processed data (in standard csv format)
+--sum_data=<sum_data>   Path to the file containing the summary data (in standard csv format)
```

Comment:
We don't need the `--sum_data=<sum_data>` argument in this script anymore since we moved the summary statistics table to the third script.